common/djangoapps/student/models.py

```diff
@@ -884,7 +913,8 @@ def is_course_full(self, course):
         """
         is_course_full = False
         if course.max_student_enrollments_allowed is not None:
-            is_course_full = self.num_enrolled_in(course.id) >= course.max_student_enrollments_allowed
+            is_course_full = self.num_enrolled_in_exclude_admins(course.id) >= course.max_student_enrollments_allowed
```

Comment:
This change will affect non-CCX courses also. Is that intended? Did this change cause any other tests to fail? If not, I fear that we're not properly testing the enrollment limits. Looks like you added course enrollment number tests below - do those tests test both CCX and non-CCX courses? Do they need to?